INFO 19:48:35  当前执行的appium相关配置为：{'platformName': 'Android', 'platformVersion': '10', 'deviceName': 'CLB0219123002349', 'udid': 'CLB0219123002349', 'systemPort': 8201, 'app': '/Users/likunyu/project/521-uiautomation/data/apk/jumei.apk', 'clearSystemFiles': True, 'appActivity': 'com.jm.android.jumei.home.activity.StartActivity', 'appPackage': 'com.jm.android.jumei', 'automationName': 'UIAutomator2', 'noSign': True, 'recreateChromeDriverSessions': True, 'unicodeKeyboard': True, 'noReset': True, 'fullReset': False, 'newCommandTimeout': 300} 

INFO 19:48:35  Could not connect to port 4723 on host localhost 

INFO 19:48:35  Could not get IP address for host: localhost 

WARNING 19:48:35  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10d9f88b0>: Failed to establish a new connection: [Errno 61] Connection refused')': /wd/hub/session 

WARNING 19:48:35  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10d9f84f0>: Failed to establish a new connection: [Errno 61] Connection refused')': /wd/hub/session 

WARNING 19:48:35  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10d9f8340>: Failed to establish a new connection: [Errno 61] Connection refused')': /wd/hub/session